templates/mwe.md

```diff
@@ -21,26 +9,30 @@ If you need to link to Lean 3 specific resources please link there.
 
 ## tl;dr
 
-When posting code on Zulip, please include all `import`s and `open`s and `universe`s and `variable`s, so others can simply just cut and paste what you post, and see the same issue that you are seeing. The best way to ensure you have done this is to make a completely new Lean file containing only what you are proposing to post, and checking that it compiles.
+When posting code on Zulip, please include all `import`s, `open`s, `universe`s, and `variable`s, so others can simply just copy-paste what you post, and see the same issue that you are seeing.
+
+The best way to ensure you have done this is to copy-paste the code snipped you are about to post into the [lean web editor](https://lean.math.hhu.de) (or an empty Lean file) and check it compiles.
+
+## Examples
 
 ### Bad example:
 
 ```lean
-#check (univ : set X)
+#check (univ : Set X)
 ```
 
 ### Good example:
 
 ```lean
-import data.set.basic
+import Mathlib
```

Comment:
Why importing everything?

Reply:
Simplicity. I decided to add the instruction below that `import Mathlib` is perfectly fine in a mwe. My reasons were the following:

- wasn't possible in lean3 (unless you created the all.lean manually first), which might be why it wasn't used.
- is much simpler for beginners than searching the correct import
- ensures that you are not just simply missing imports (e.g. if some instances are not in `XY/Basic.lean`)
- Recent conversations on Zulip do use `import Mathlib` on a frequent basis in the MWEs.